src/Common/tests/Helpers/WebHostExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Hosting.Server.Features;
+
+namespace Microsoft.AspNetCore.Hosting
+{
+    public static class WebHostExtensions
+    {
+        public static int GetHttpPort(this IWebHost webHost)
```

Comment:
Could this be implemented as
```c#
public static int GetHttpPort(this IWebHost webHost) => GetHttpBaseAddressUri(webHost).Port;
```
Similar idea for https

Reply:
done